File: swift/InkRecognition/Recognizer/Recognizer/InkBullet.swift

import Foundation

@objc
class InkBullet : InkRecognitionUnit {
    
    var text:String!
    
    @objc
    override init(json : [String:Any]) {
        self.text = json["recognizedText"] as? String ?? ""
        super.init(json:json)
    }
}

File: java/InkRecognition/android-sample-app/Recognizer/app/src/main/java/CognitiveServices/Ink/Recognition/InkBullet.java
package CognitiveServices.Ink.Recognition;

import android.util.DisplayMetrics;
import org.json.JSONException;
import org.json.JSONObject;

class InkBullet extends InkRecognitionUnit {

    public String getText() {
        return text;
    }

    private String text;

    InkBullet(String bulletJSON, DisplayMetrics metrics, InkRoot result) throws JSONException {
        super(bulletJSON, metrics, result);
        JSONObject jsonBullet = new JSONObject(bulletJSON);
        this.text = jsonBullet.getString("recognizedText");
    }
}
